src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlElasticPoolActivity.md

```diff
@@ -64,10 +64,23 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -OperationId
+The ID of the operation to retrieve.
+Type: Guid
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+
+
 ### -ResourceGroupName
 Specifies the name of a resource group to which the elastic pool is assigned.
 
-```yaml
```

Comment:
not sure why this tag got removed in a lot of places... Please regenerate or revert and regenerate, as the file now is formatted incorrectly (see "view")